docs/concepts/get-offline-features.md

```diff
@@ -71,11 +71,25 @@ The path of a dataset as the 'spine' for the to-be-created training dataset. We
 2. **Timestamp Column:** A column representing the event time of the row. By default, Feathr will make sure the feature values queried have a timestamp earlier than the timestamp in observation data, ensuring no data leakage in the resulting training dataset. Refer to [Point in time Joins](./point-in-time-join.md) for more details.
 
 3. **Other columns** will be simply pass through to the output training dataset, which can be treated as immutable columns.
-
 ## More on `Feature Query`
 
 After you have defined all the features, you probably don't want to use all of them in this particular program. In this case, instead of putting every features in this `FeatureQuery` part, you can just put a selected list of features. Note that they have to be of the same key.
 
+## Feature names conflicts check 
```

Comment:
Feature naming conflicts? `names` sounds weird...

Reply:
Will modify it in another doc related PR